src/Sdk/PasswordlessClient.cs

```diff
@@ -41,6 +39,16 @@ public PasswordlessClient(HttpClient client)
         _client = client;
     }
 
+    /// <inheritdoc/>
+    public async Task AddAliasAsync(AddAliasRequest request, CancellationToken cancellationToken)

```

Comment:
Since AddAlias might also mean remove aliases, do we want to name it `SetAlias` instead?

Reply:
@abergs Would it make sense to split the endpoints for adding and removing aliases?